astronomer/airflow/version_check/update_checks.py

```diff
@@ -343,11 +398,7 @@ class UpdateAvailable(BaseApi):
         allow_browser_login = True
 
         @expose("<path:version>/dismiss", methods=["POST"])
-        @auth.has_access(
-            [
-                ("can_dismiss", 'UpdateAvailable'),
-            ]
-        )
+        @has_access(method="can_dismiss", resource_type="UpdateAvailable")
```

Comment:
How about the new function's signature matching the old one? That would fix the CI

Reply:
Sorry didn't test this earlier, thinking small change, did it this time around. Thanks for pointing this out.